packages/blade/src/components/FileUpload/FileUpload.web.tsx

```diff
@@ -377,11 +372,8 @@ const _FileUpload: React.ForwardRefRenderFunction<BladeElementRef, FileUploadPro
               size={size}
               onRemove={() => {
                 const newFiles = selectedFiles.filter(({ id }) => id !== file.id);
-                flushSync(() => {
```

Comment:
We are also removing flushSync?

Reply:
Yup we added it to fix the onChange args right? now we are not calling onChange on remove